On boarding/nahuel gil/my trello app

NahuelGil98 opened the pull request.

Commit:
#Feature: Card Module

Commit:
#Feature: Card Module

Commit:
#Feature: Delete card function

Commit:
#FIX: Fix when I press "No Thanks" button.

rjulianialtimetrik commented on src/app/components/board-component/board-component.component.html:
```diff
@@ -0,0 +1,68 @@
+  <div>
+  </div>
+  <div>
+<!--    <mat-grid-list cols="4" rowHeight="2:2">-->
```
Please simply remove the unused lines of code.

NahuelGil98 replied:
Done, Thanks !

rjulianialtimetrik commented on src/app/components/card-component/card-component.component.ts:
```diff
@@ -0,0 +1,53 @@
+  styleUrls: ['./card-component.component.css']
+})
+export class CardComponentComponent implements OnInit {
+  @Input() title: string;
```
It's better if you define a model class, let's say, "Ticket" or "BoardItem" that contains your item's attributes. Then, you bind your inputs to said attributes in an instance of the model. makes up for clearer, self-contained and more reusable code.

NahuelGil98 replied:
Perfect, Thanks!

rjulianialtimetrik commented on src/app/components/card-component/card-component.component.ts:
```diff
@@ -0,0 +1,53 @@
+      console.log(result);
+      // tslint:disable-next-line:triple-equals
+      if (result != 'undefined' || result != '') {
+        console.log(`Result : ${result} - cardId ${cardId} - tabId ${tabId}`);
```
might be a good idea to remove all console.log statements before pushing to the repo.

NahuelGil98 replied:
Done, thanks!

rjulianialtimetrik commented on src/app/components/card-dialog/card-dialog.component.ts:
```diff
@@ -0,0 +1,30 @@
+  styleUrls: ['./card-dialog.component.css']
+})
+export class CardDialogComponent implements OnInit {
+  cardTitle = '';
```
Same as before, please define a "Card" model class and use it for your inputs.

rjulianialtimetrik commented on src/app/components/column-comoponent/column-comoponent.component.html:
```diff
@@ -0,0 +1 @@
+<p>column-comoponent works!</p>
```
Is this component supposed to be used in your app at all? if not, please remove it.

rjulianialtimetrik commented on src/app/services/tabs.service.ts:
```diff
@@ -0,0 +1,90 @@
+  addTab(tabTitle) {
+  console.log(` tab title: ${tabTitle}`);
+  if (tabTitle !== '' || tabTitle === undefined) {
+      this.newTab = {
```
It's better to have a model class for tab

NahuelGil98 replied:
Ok, Thanks

rjulianialtimetrik commented on src/app/services/tabs.service.ts:
```diff
@@ -0,0 +1,90 @@
+// Add a card to a tab
+  addCard(card, tabId) {
+    this.newCard = {
+      id: uuid(),
```
it's better to have a model class for card

NahuelGil98 replied:
Ok, Thanks!

Commit:
#FIX:
- I create tab and card classes to make the code more clearer, self-contained and reusable.
- I cleared some unused code.

- console.log(); cleared.

Commit:
#FIX: - console.table(); cleared.

Commit:
#FIX: - Fixing some ids
